lib/awspec/type/rds.rb

```diff
@@ -51,6 +51,12 @@ def has_option_group?(name)
       end
     end
 
+    def has_tag?(tag_key, tag_value)
+      tag = find_rds_tags(@display_name, tag_key)
```

Comment:
`@display_name` is db_instance_identifier ( neither  name nor arn ), now.
( see https://github.com/k1LoW/awspec/blob/master/lib/awspec/helper/finder/rds.rb#L4 )

So, I  think So, the begin~end of `find_rds_tags` is not needed.